tests/boulder-start.sh

```diff
@@ -4,11 +4,16 @@
 
 # ugh, go version output is like:
 # go version go1.4.2 linux/amd64
-GOVER=`go version | cut -d" " -f3 | cut -do -f2`  
+GOVER=`go version | cut -d" " -f3 | cut -do -f2`
 
 # version comparison
 function verlte {
+  if [ `uname` == 'Darwin' ]; then
+    [ "$1" = "`echo -e \"$1\n$2\" |  sed 's/\b\([0-9]\)\b/0\1/g' \
+      | sort | sed 's/\b0\([0-9]\)/\1/g' | head -n1`" ]
```

Comment:
this is so complicated that it's worth to put any inline comment, at least "`sort -V` doesn't work on Mac"